lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_update_metadata.rb

```diff
@@ -1,52 +0,0 @@
-module Fastlane
-  module Actions
-    module SharedValues
-      ANDROID_UPDATE_METADATA_CUSTOM_VALUE = :ANDROID_UPDATE_METADATA_CUSTOM_VALUE
-    end
-
-    class AndroidUpdateMetadataAction < Action
-      def self.run(params)
-        require_relative '../../helper/android/android_git_helper'
-
-        Fastlane::Helper::Android::GitHelper.update_metadata(ENV['validate_translations'])
```

Comment:
The corresponding method in the `android_git_helper.rb` should also be removed in parallel to removing the action.

Note that this helper method is also marked as `@todo` so the fact that you missed it makes me wonder if you used case-sensitive search when looking for TODOs/FIXMEs. We should definitively try to be consistent in using `@TODO`/`@FIXME` in all uppercase everywhere (I _think_ that's the ruby convention?) but 🤷‍♂️ 

---

PS: As a bonus it would be nice to remove the `./tools/update-translations.sh` script (which is the script the `update_matadata` helper ends up calling) from client repos in parallel to that PR

Reply:
> As a bonus it would be nice to remove the `./tools/update-translations.sh` script 

Bonus points to @JavonDavis for doing that a while back:

- https://github.com/woocommerce/woocommerce-android/commit/7fec286c18#diff-0baa79b5bfd3b41d3e2b326f974f71a14a7c91b4a0b01bad6c55ef882912f64f
- https://github.com/wordpress-mobile/WordPress-android/commit/28af2e3823#diff-0baa79b5bfd3b41d3e2b326f974f71a14a7c91b4a0b01bad6c55ef882912f64f